docs/source/get_involved/contributing.rst

```diff
@@ -223,17 +223,11 @@ We use `pytest <https://docs.pytest.org/en/latest/>`__ for unit testing.
 To check if your code passes all tests locally, you need to install the
 development version of sktime and all extra dependencies.
 
-1. Install all extra requirements from the root directory of sktime:
+1. Install the development version of sktime:
```

Comment:
perhaps more precisely: "install the development version of sktime, with all developer dependencies"?

Similarly, should the dependency sets be listed anywhere in the install instructions?

Reply:
There are not all developer dependencies since also docs can be considered dev dependencies, but I've updated the line.